.travis.yml

```diff
@@ -27,18 +19,44 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported PHP and Symfony version
+        - php: 7.1
+          env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="composer test-ci"
```

Comment:
should we actually run the coverage on the min deps ?

Reply:
i'd do coverage on the latest stable